tests/fixtures/ts-config-files/helper.ts

```diff
@@ -0,0 +1,44 @@
+export type RuleLevelAndOptions<Options extends any[] = any[]> = Prepend<
```

Comment:
Can you add a comment to the top of this file explaining its purpose?

Reply:
I tried to keep it short, is [this](https://github.com/eslint/eslint/blob/479847a7ad74a9cca65502b6d992347ad49e7873/tests/fixtures/ts-config-files/helper.ts#L1-L5) okay?

Reply:
Looks good. 👍